tests/expected/intrinsics/copy-nonoverlapping/copy-overflow/main.rs

```diff
@@ -1,8 +1,14 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-
-// Checks that `copy_nonoverlapping` triggers an overflow failure if the `count`
-// argument can overflow a `usize`
+// kani-flags: --legacy-linker
+//! Checks that `copy_nonoverlapping` triggers an overflow failure if the `count`
+//! argument can overflow a `usize`
+//!
+//! When enabling "--mir-linker", the error we currently get is:
+//! > Failed checks: Called `Option::unwrap()` on a `None` value
+//! This happens because of std debug checks. The `is_nonoverlapping` check has a
```

Comment:
Honestly I wonder if we should just disable `debug_assert` in our build of `std` for now?

Reply:
That's an interesting idea. It would help with this for now. It would be interesting to see how much more UB checks the std encode as `debug_assert()`.